contracts/passThrough/PassThrough.sol

```diff
@@ -33,13 +36,13 @@ contract PassThrough is Ownable, PassThroughStorage {
     */
     function() external {
         require(
-            isOwner() || (isOperator() && isMethodAllowed(msg.sig)),
+            isOperator() && isMethodAllowed(msg.sig) || isOwner(),
```

Comment:
Warning! I need to check precedence rules to be sure if this isn't lazy-evaluated from left-to-right (I'm not sure if `false && true || true` is `false && (true || true)` or `(false && true) || true`).

Reply:
I test it on remix and `&` has more precedence 

```solidity
 function checkLogic() public view returns (bool a, bool b, bool c, bool d, bool e) {
        a = false && true || true; // true
        b = true && false || true; // true
        c = true && true || false; // true
        d = false && false || true; // true
        e = false && false || false; // false
    }
```